Modules/Module_Processing/Module_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -203,47 +203,46 @@ function xASL_wrp_ResampleASL(x)
 
 
 %% ------------------------------------------------------------------------------------------
-% 8. Pair-wise subtraction & saving PWI & PWI4D in both spaces
+% 8. Pair-wise control-label subtraction
+% for both native & standard space
+
+% Nomenclature:
+% ASL4D = original timeseries
+% PWI4D = subtracted timeseries -> for quantification purposes
+% PWI3D = averaged per TE—PLD—LabDur combination -> for QC purposes
+% PWI   = single perfusion-weighted volume -> for registration purposes
+
+% 
+
 % Load ASL time series (after being pre-processed)
 StringSpaceIs = {'native' 'standard'};
-PathASL = {x.P.Path_rdespiked_ASL4D x.P.Path_rtemp_despiked_ASL4D};
+PathASL4D = {x.P.Path_rdespiked_ASL4D x.P.Path_rtemp_despiked_ASL4D};

```

Comment:
Really? Cause the first is native space and second population space. For all the things below. But this doesn't seem to be correct here.

Reply:
This one only exists in native space. We never use it in standard space, only the intermediate files (like PWI4D) we store and use in standard space